tests/test_logging.py

```diff
@@ -409,3 +410,48 @@ def test_server_documentation_conformance(inmanta_config, monkeypatch):
     )
 
     assert from_config._to_dict_config() == from_file_dict
+
+
+async def test_print_default_logging_cmd(inmanta_config, tmp_path):
+    """
+    Test that piping to file does not change the logging config
+    """
+    components = ["scheduler", "server", "compiler"]
+    for component in components:
+        args = [sys.executable, "-m", "inmanta.app", "print-default-logging-config", component]
+
+        # Output the logging config on the CLI.
+        # Here we force ENVIRON_FORCE_TTY to be set to simulate that we are on a TTY
+        process = await subprocess.create_subprocess_exec(
+            *args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env={ENVIRON_FORCE_TTY: "yes"}
+        )
+        await process.wait()
```

Comment:
As the documentation mentions, `process.wait()` can deadlock when `subprocess.PIPE` is used. I don't really see why this call is required, given that we already call into `process.communicate()`.